### DISCUSSION TITLE
Efficiency dependent on input power

### DISCUSSION THREAD

**Discussion by bruno-luca** [PyPSA/pypsa-eur#1378] [Q&A]
Good morning all, 
I am stuck on a part of the project I am working on. 
I have an electrolyzer that produces hydrogen. I've defined the electrolyzer as Link, which takes in input electricity and converts it to hydrogen. The electrolyzer's efficiency varies depending on the input power. From what I understand, it's not possible to define the efficiency of the electrolyzer as dependent on input power. Is there any way to achieve this? 
Thanks in advance to anyone who can help me out :)

**Comment by fneum** [comment]
No, it's not possible in PyPSA due to the non-linearity introduced by coupling $\eta(p) * p$.
